ProcessJsonControl.Designer.cs

```diff
@@ -21,246 +20,132 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
-        #region Vom Komponenten-Designer generierter Code
+        #region Component Designer generated code
 
         /// <summary> 
-        /// Erforderliche Methode für die Designerunterstützung. 
-        /// Der Inhalt der Methode darf nicht mit dem Code-Editor geändert werden.
+        /// Required method for Designer support - do not modify 
+        /// the contents of this method with the code editor.
```

Comment:
What happened with ``InitializeComponent``? It doesn't look like this was edited via the graphical/GUI editor. 

See the comment you translated:

```
/// Required method for Designer support - do not modify 
/// the contents of this method with the code editor.
```